docs/config.md

```diff
@@ -45,6 +47,20 @@ Salesforce uses the OAuth protocol to allow application users to securely access
     5. From the response that you get, extract the access token to access Salesforce via the created application. 
        > NOTE: You will also get a refresh token to renew the access token when it expires.
 
+### Importing the Salesforce Certificate
+
+Before you start configuring the connector, import the Salesforce certificate to your EI client keystore.
+
+* Follow the steps below to import the Salesforce certificate into the EI client keystore:
+
+    1. To view the certificate, log in to your Salesforce account in your browser (e.g., https://login.salesforce.com), and click the lock on the address bar.
```

Comment:
To view the certificate, log in to your Salesforce account using your browser (e.g., https://login.salesforce.com), and click the lock icon on the address bar.

Reply:
Fixed in a0448a990213d3cdb3f3ae158ef2307753bad900